src/formats/mmCIF.cpp

```diff
@@ -91,6 +93,16 @@ mmCIFFormat::mmCIFFormat(std::string path, File::Mode mode, File::Compression co
             gamma = cif_to_double(line_split[1]);
         }
 
+        if (line_split[0] == "_entry.id") {
+            pdb_idcode_ = line_split[1];
+        }
+
+        if (line_split[0] == "_struct.title") {
+            auto temp_name = trim(line.substr(13));
+            std::copy(temp_name.begin() + 1, temp_name.end() - 1,
```

Comment:
Why the std::copy here, instead of `name_ = temp_name.substr(1, temp_name.size() - 2)` ?

Reply:
That's fine, but I don't like to do subtraction on 'size()' because there's the chance that it is zero and the result explodes. I'll add a check as well.